### PR TITLE
[Feature][doc] Add a guide to nominate new committer

### LINKED ISSUE #10006: [Feature][doc] Add a guide to nominate new committer
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

we could learning from https://seatunnel.apache.org/community/contribution_guide/committer#project-management-committee

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by TyrantLucifer**
## Purpose of the pull request

close #10006

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: docs/docs/zh/contribute/join/become-a-committer.md [hunk 1]
```diff
@@ -1,12 +1,159 @@
...
 
-在Dolphinscheduler社区，如果一个committer获得大量的优秀成绩，就可以被邀请加入项目管理委员会（PMC）。
+成为committer的第一步是要学习如何为Dolphinscheduler贡献并开始为Dolphinscheduler贡献，任何人可以向项目提交补丁、使用文档、测试用例等。
 
-当您不熟悉ASF使用的开源的开发过程时，有时难以理解的一点，就是我们更重视社区而不是代码。一个强大而健康的社区将受到尊重，成为一个有趣和有益的地方。更重要的是，一个多元化和健康的社区
-可以长时间的持续支持代码，即使个别公司在这个领域来来往往，也是如此。
+PMC会定期基于活跃贡献者们对Dolphinscheduler的贡献从他们中提名增加新的committer，要被提名为committer的条件如下：
 
-更多详细信息可以在[这里](https://community.apache.org/contributors/)找到
+1. 对Dolphinscheduler有持续的贡献：候选人应该对Dolphinscheduler有重大贡献，在整个项目中做出广泛的贡献，并且至少贡献了一个主要的组件，并在其中扮演了 "owner"的角色。
```
Comment (EricGao888) [inline docs/docs/zh/contribute/join/become-a-committer.md]:
```suggestion
1. 对Dolphinscheduler有持续的贡献：候选人应该对Dolphinscheduler有重大贡献，在整个项目中做出广泛的贡献，并且至少贡献了一个主要的组件，并在其中扮演了"owner"的角色。
```

File: docs/docs/zh/contribute/join/become-a-committer.md [hunk 2]
```diff
@@ -1,12 +1,159 @@
...
+
+## 提名新的committer
+
+在Dolphinscheduler中，committer提名只能由现有的PMC成员开始。如果一个新的提交者觉得他/她有资格，他/她应该联系任何现有的PMC成员并进行讨论。如果这一点在PMC的一些成员中得到了认同，那么这个过程就会启动。
+
+建议采取以下步骤（仅需要现有2个PMC成员就可以发起）：
+1. 发送一封标题为"[讨论]晋升xxx为新的committer"的电子邮件到`private@dolphinscheduler.apache.org`，并在邮件中列出该候选人的重要贡献，这样就可以提名流程。
+2. 保持讨论要超过3天，但不超过1周，除非有任何明确的反对。
+3. 如果PMC普遍同意该提议，请发送一封题为"[投票]提名xxx成为新的committer"的电子邮件到`private@dolphinscheduler.apache.org`
+4. 保持投票过程超过3天，但不超过1周。如果有三张+1票，且+1票>-1票，则视为 "共识批准"。注：+1票 > -1票。
```
Comment (EricGao888) [inline docs/docs/zh/contribute/join/become-a-committer.md]: A positive result is at least 3 + 1 votes with **NO** vetoes, see: https://community.apache.org/newcommitter.html

File: docs/docs/en/contribute/join/become-a-committer.md [hunk 3]
```diff
@@ -1,11 +1,170 @@
...
+## Nominate new committer
+
+In Dolphinscheduler, **new committer nomination** could only be officially started by existing PMC members. If a new committer feels that he/she is qualified, he/she should contact any existing PMC member and discuss. If this is agreed among some members of the PMC, the process will kick off.
+
+The following steps are recommended (to be initiated only by an existing 2PMC member):
+1. Send an email titled `[DISCUSS] Promote xxx as new committer` to `private@dolphinscheduler.apache.org`. List the important contributions of the candidate,
+   so you could gather support from other PMC members for your proposal.
+2. Keep the discussion open for more than 3 days but no more than 1 week, unless there is any express objection or concern.
+3. If the PMC generally agrees to the proposal, send an email titled `[VOTE] Promote xxx as new committer` to `private@dolphinscheduler.apache.org`.
+4. Keep the voting process open for more than 3 days, but no more than 1 week. Consider the result as `Consensus Approval` if there are three +1 votes and
```
Comment (EricGao888) [inline docs/docs/en/contribute/join/become-a-committer.md]: A positive result is at least 3 + 1 votes with NO vetoes, see: https://community.apache.org/newcommitter.html

**Review by EricGao888 — DISMISSED** [review]
Overall LGTM except few nits.

File: docs/docs/en/contribute/join/become-a-committer.md (hunk 3 above)
Comment (TyrantLucifer) [reply docs/docs/en/contribute/join/become-a-committer.md]: Resolved.

File: docs/docs/zh/contribute/join/become-a-committer.md (hunk 2 above)
Comment (TyrantLucifer) [reply docs/docs/zh/contribute/join/become-a-committer.md]: Resolved.

File: docs/docs/zh/contribute/join/become-a-committer.md (hunk 1 above)
Comment (TyrantLucifer) [reply docs/docs/zh/contribute/join/become-a-committer.md]: Resolved.

**Review by EricGao888 — DISMISSED** [review]

**Review by EricGao888 — DISMISSED** [review]

**Comment by EricGao888** [comment]
Seems there are dead links.

**Comment by TyrantLucifer** [comment]
> Seems there are dead links.

Waiting for me to confirm.

**Comment by EricGao888** [comment]
> > Seems there are dead links.
> 
> Waiting for me to confirm.

image

**Comment by TyrantLucifer** [comment]
> > > Seems there are dead links.
> > 
> > 
> > Waiting for me to confirm.
> 
> image



> > > Seems there are dead links.
> > 
> > 
> > Waiting for me to confirm.
> 
> image



> > > Seems there are dead links.
> > 
> > 
> > Waiting for me to confirm.
> 
> image

Fixed, thx for your review, let's waiting CI.

**Comment by TyrantLucifer** [comment]
> > > Seems there are dead links.
> > 
> > 
> > Waiting for me to confirm.
> 
> image

Fixed, thx for your review, let's waiting CI.

File: docs/docs/en/contribute/join/become-a-committer.md [hunk 4]
```diff
@@ -1,11 +1,168 @@
...
+
+Community involvement: Committers should have a constructive and friendly attitude in all community interactions. They should also be active on the dev and user list and help mentor newer contributors and users. In design discussions, committers should maintain a professional and diplomatic approach, even in the face of disagreement.
+
+## Nominate new committer
+
+In Dolphinscheduler, **new committer nomination** could only be officially started by existing PMC members. If a new committer feels that he/she is qualified, he/she should contact any existing PMC member and discuss. If this is agreed among some members of the PMC, the process will kick off.
+
+The following steps are recommended (to be initiated only by an existing 2PMC member):
```
Comment (CalvinKirs) [inline docs/docs/en/contribute/join/become-a-committer.md]:
```suggestion
The following steps are recommended (to be initiated only by an existing PMC member):
```

**Review by CalvinKirs — DISMISSED** [review]

**Review by CalvinKirs — APPROVED** [review]

**Comment by zhongjiajie** [comment]
Thanks all

**Pull request merged**